public/samples/DataStreams/StreamsUpkeep.sol

```diff
@@ -110,65 +98,48 @@ contract StreamsUpkeep is ILogAutomation, StreamsLookupCompatibleInterface {
         return (true, abi.encode(values, extraData));
     }
 
-    function performUpkeep(bytes calldata performData) external override {
+    // function will be performed on-chain
+    function performUpkeep(bytes calldata performData) external {
+        // Decode incoming performData
```

Comment:
// Decode incoming performData

maybe //decode the performData bytes passed in by CL Automation. This contains the data stream reports.

Reply:
Done